src/Lithnet.GoogleApps/GroupRequestFactory.cs

```diff
@@ -334,7 +337,7 @@ public IEnumerable<string> GetAliases(string id)
             {
                 GroupsResource.AliasesResource.ListRequest request = connection.Item.Groups.Aliases.List(id);
                 Aliases aliases = request.ExecuteWithRetryOnBackoff();
-                return aliases?.AliasesValue?.Select(t => t.AliasValue) ?? new List<string>();
+                return aliases.AliasesValue?.Select(t => t.AliasValue) ?? new List<string>();
```

Comment:
I'm sure I've had aliases return null before. Can we leave the null operator in there. I'd rather not risk deploying to production and encountering a random NullReferenceException

Reply:
Mistake on my side, missed updating the file. Changing back.